.github/workflows/check-generated-files.yml

```diff
@@ -62,7 +62,7 @@ jobs:
       - name: Set up protoc
         if: steps.changes.outputs.proto == 'true'
         run: |
-          PROTOC_VERSION=25.1
+          PROTOC_VERSION=25.2
```

Comment:
Update `PROTOC_GEN_VERSION` to `v1.32.0`.

Reply:
The latest version is `25.3` https://github.com/protocolbuffers/protobuf/releases/tag/v25.3

Reply:
I am talking about `protoc-gen` plugin for Go: [https://github.com/protocolbuffers/protobuf-go](https://github.com/protocolbuffers/protobuf-go/releases)